### DISCUSSION TITLE
Upgrade to graphql-java-extended-scalars 20.0

### DISCUSSION THREAD

**Discussion by jandrade1** [Netflix/dgs-framework#1379] [General]
Hi,

I'm interested in using the new graphql-java-extended-scalars 20.0 dependency for the CountryCodeScalar & CurrencyScalar they've recently added. Is there a way to upgrade that on my own application? I believe the Netflix DGS BOM enforces the version to be less than 20.0, and while maybe I can try to force the version override, I'm not sure if it can be done safely. Thanks!

**Comment by srinivasankavitha** [comment]
The BOM ensures that the versions of libraries are compatible with each other. You could try overriding the version be adding the dependency explicitly and try it. If your app works,  it should be good enough.

**Reply by jandrade1** [reply]
Thanks @srinivasankavitha , I got it to work. If anyone is curious, I'm doing something like this (using the Spring Dependency Management plugin):

```
dependencyManagement {
    imports {
        mavenBom("com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:${netflixDgsVersion}")
    }
    dependencies {
        dependency 'com.graphql-java:graphql-java-extended-scalars:20.0'
    }
}
```

Of course any additional scalars need to be registered and configured for the codegen plugin.
